src/button.tsx

```diff
@@ -433,7 +433,7 @@ interface ButtonLinkCommonProps {
     trackEvent?: boolean;
     /** "data-" prefix is automatically added. For example, use "testid" instead of "data-testid" */
     dataAttributes?: DataAttributes;
-    aligned?: boolean;
+    aligned?: boolean | 'left' | 'right';
```

Comment:
don't you prefer having 2 props? `leftAligned` and `rightAligned` and deprecate the current `aligned`

Reply:
Does it make sense to align it to both sides?

Reply:
@atabel as Pedro says, it makes no sense to align on both sides. I did it this way to avoid a breaking change for the buttons that are currently using the prop with value = true (in order to have backward compatibility).

Reply:
Discussed offline. We deprecated this prop and  introduced new ones that allow adding bleed to left, right and top+bottom